dts/bindings/adc/adi,ad7124-adc.yaml

```
@@ -17,6 +17,17 @@ properties:
       - true: Enable internal reference
       - false: Disable internal reference
 
+  pd-switch-enable:
+    type: boolean
+    default: false
+    description: |
+      Enable internal low side power switch for AD7124
+      - true: The switch is in the open position
+      - false: The switched is in the closed position
+      The default configuration places the switch in the
+      closed position for control setup upon power-up 
+      or after reset.
```

```suggestion
    description: |
      Enable internal low-side power-down switch for AD7124
      - true: The switch is in the closed position
      - false: The switch is in the open position
      The default configuration places the switch in the
      open position for control setup upon power-up 
      or after reset.
```
Is the open/close logic inverted?